ci-operator/config/bentitopolis/operator-metering/bentitopolis-operator-metering-master.yaml

```diff
@@ -0,0 +1,73 @@
+base_images:
+  ansible-operator:
+    cluster: https://api.ci.openshift.org
+    name: "4.5"
+    namespace: ocp
+    tag: ansible-operator
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.5"
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    context_dir: .
+    dockerfile_path: Dockerfile.src
+images:
+- dockerfile_path: Dockerfile.reporting-operator.okd
+  from: base
+  to: metering-reporting-operator
+- dockerfile_path: Dockerfile.metering-ansible-operator
+  from: ansible-operator
+  to: metering-ansible-operator
+promotion:
+  name: "4.5"
+  namespace: ocp
+resources:
+  '*':
+    limits: {}
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: "4.5"
+  namespace: ocp
+tests:
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: make unit TEST_OUTPUT_DIR=/tmp/artifacts
+  container:
+    from: src
+- as: verify
+  commands: |
+    set -x
+    set -e
+    # (chance): Hacks to allow us write access to our source files since
+    # they're 664 in the image and owned by root:root, and we get a random UID
+    # in Openshift pods which means we don't have write permissions.
+
+    # create a copy of the source so we can modify the files.
+    export GOPATH=/tmp/go
+    mkdir -p $GOPATH/src/github.com/bentitopolis/
+    cp -r /go/src/github.com/bentitopolis/operator-metering/ $GOPATH/src/github.com/bentitopolis/operator-metering
+    cd $GOPATH/src/github.com/bentitopolis/operator-metering
+
+    # Run the actual tests that nothing in the changes after re-generating
+    # files. This ensures generated files aren't modified directly.
+    make verify
```

Comment:
```suggestion
    make verify GO_PKG=github.com/bentitopolis/operator-metering
```

Reply:
We would want to do this for the release-4.5 and master branch configurations too.